cvat/apps/engine/media_extractors.py

```diff
@@ -658,7 +658,7 @@ def save_as_chunk(self, images, chunk_path):
                         if has_exif_rotation(pil_image):
                             rot_image = ImageOps.exif_transpose(pil_image)
                             try:
-                                if rot_image.format == 'TIFF':
+                                if pil_image.format == 'TIFF':
```

Comment:
rot_image.format is None after exif_transpose 
It results to jpeg image saved with .tif filename